packages/react-app/src/lib/constants.js

```diff
@@ -60,35 +60,44 @@ export const networkCurrencies = {
   },
 };
 
-const INFURA_ID = process.env.REACT_APP_INFURA_ID;
+const { REACT_APP_INFURA_ID: INFURA_ID } = process.env;
 
 export const chainUrls = {
   1: {
-    rpc: `https://mainnet.infura.io/v3/${INFURA_ID}`,
+    rpc: [
+      `https://mainnet.infura.io/v3/${INFURA_ID}`,
+      `https://mainnet-nethermind.blockscout.com/`,
```

Comment:
May the list of URLs still be provided through the environment variables rather even if there are hardcoded values?

Reply:
You mean the fallback URLs alone @akolotov?

Reply:
No, my suggestion is to allow variables `REACT_APP_HOME_RPC_URL` and `REACT_APP_FOREIGN_RPC_URL` to keep several URLs divided by space. The first URL in the list is considered as the main RPC endpoint. The rest URLs in the list (it could be more than one URL) are considered as fallback RPC endpoints. The main RPC endpoint and fallback endpoints will be accessed in round robin manner: if one fails the next in the list is being tried. We have a similar functionality in the bridge oracles.

Reply:
As far as Round robin pattern is concerned, that's exactly what has been implemented. But I am not sure what you mean by dividing the space. @akolotov ..

Reply:
<img width="1790" alt="Screenshot 2021-04-02 at 2 47 09 AM" src="https://user-images.githubusercontent.com/32637757/113355010-bbfb9200-935d-11eb-84ae-04863f02d4b2.png">
That's weird, coz it's instructing at least for me to switch networks before proceeding .. 
@dan13ram, I guess it's due to the reversing logic?

Reply:
@sunguru98 What he means is to let the env variables `REACT_APP_HOME_RPC_URL` and `REACT_APP_FOREIGN_RPC_URL` store the rpc urls separated by space. 
So that the javascript can intepret them by reading and splitting by ` ` character. 
Hope that makes sense.

Reply:
Gotcha.